pkgs/applications/audio/easyaudiosync/default.nix

```diff
@@ -0,0 +1,98 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, cmake
+, qtbase
+, qttools
+, spdlog
+, ffmpeg
+, taglib
+, wrapQtAppsHook
+, makeDesktopItem
+, copyDesktopItems
+,
+}:
+let
+  isDarwin = stdenv.isDarwin;
+  isLinux = stdenv.isLinux;
+in
+stdenv.mkDerivation rec {
+  pname = "easyaudiosync";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "complexlogic";
+    repo = "EasyAudioSync";
+    rev = "v${version}";
+    hash = "sha256-w98tj9BuixPhuDgwn74EYY0gvKH6kbfQmtg030RWRU0=";
+  };
+
+  patches =
+    [
+      ./0001-fix-project-name.patch
+      ./0002-fix-qt67-deprecated-methods.patch
+      ./0004-force-qt6.patch
+    ]
+    ++ lib.optionals isDarwin [
```

Comment:
Can we apply this patch unconditional? Otherwise in an update we might not notice that the patch no longer applies